src/test/logical_query_plan/lqp_translator_test.cpp

```diff
@@ -964,4 +969,24 @@ TEST_F(LQPTranslatorTest, CreatePreparedPlan) {
   EXPECT_EQ(prepare->prepared_plan(), prepared_plan);
 }
 
+TEST_F(LQPTranslatorTest, Export) {
+  const auto lqp = ExportNode::make("a_table", "a_file.tbl", FileType::Auto, ValidateNode::make(int_float_node));
```

Comment:
Please have a look at other LQP tests regarding the formatting. ValidateNode should be on its own line and indented.